app/src/main/java/com/eventyay/organizer/core/auth/start/StartFragment.java

```diff
@@ -46,24 +50,39 @@
     private StartFragmentBinding binding;
     private Validator validator;
     private SharedViewModel sharedViewModel;
+    private SharedPreferences sharedPreferences;
+    private SharedPreferences.Editor editor;
+    AlertDialog warningAlertDialog;
```

Comment:
Codacy found an issue: [Variable 'warningAlertDialog' must be private and have accessor methods.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=6423984)